pages/weeklyoverview.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import WeeklyOverview from '../components/WeeklyOverview';
+
+export default function WeeklyOverviewPage () {
+    return (
+        <WeeklyOverview />
+    );
+    }
```

Comment:
Just move the closing bracket to the start of the line.

Reply:
i made a new pull request as i had a error with my feature branch. thank you very much for your suggestions :)